src/mp4box/stsd.rs

```diff
@@ -51,6 +53,125 @@ impl Default for StsdBoxContent {
     }
 }
 
+impl StsdBoxContent {
+    /// Usually 8, but 10 for HDR (for example).
+    pub fn bit_depth(&self) -> Option<u8> {
```

Comment:
```suggestion
    /// Per color component bit depth.
    ///
    /// Usually 8, but 10 for HDR (for example).
    pub fn bit_depth(&self) -> Option<u8> {
```

I guess this _could_ expand to audio bit depth in the future making this a bit handwavy. but I guess at this point we'll have to introduce an enum [...]